weathersimulator-python/func.py

```diff
@@ -0,0 +1,33 @@
+import json
```

Comment:
The linter is fails:
```
func.py:3:1: F401 'random' imported but unused
func.py:4:1: F401 'flask.jsonify' imported but unused
func.py:22:28: E231 missing whitespace after ','
```

Reply:
I have corrected the above mentioned lint issues.